### PR TITLE
Delete pending registrations automatically - 2 bugfix #4225

### PR DESCRIPTION
**Pull request by MasterHans**
BugFix for
#4841 

Made style of label selector for checkboxes in the pending registration grid.

### CODE REVIEW

File: static/less/form.less [hunk 1]
```diff
@@ -70,6 +70,10 @@ select.form-control:not([multiple]) {
 
 label {
     font-weight: normal;
+}
+
+//.select-on-check-all:first-child
```
Comment (luke-) [inline static/less/form.less:75]: @MasterHans Why is this commented out? Shouldn't that be restricting the change?

**Review by luke- — CHANGES_REQUESTED** [review]

File: static/less/form.less (hunk 1 above)
Comment (MasterHans) [reply static/less/form.less]: This is debugging comment. Unfortunately its could not work on here

**Comment by luke-** [comment]
@MasterHans 

Is the Label CSS change only for Pending Registrations Table or are there issues here in other locations as well? 
If it is just the Pending Registration table, it might be good to give the table a class or find another unique selector.

e.g.
image

Untested:

```less
div.PendingRegistrations {
    thead > tr > th > label, tbody > tr > td > label {
        margin-bottom: 0
        height: 17px;
    }
}
```

**Comment by MasterHans** [comment]
> @MasterHans
> 
> Is the Label CSS change only for Pending Registrations Table or are there issues here in other locations as well?
> If it is just the Pending Registration table, it might be good to give the table a class or find another unique selector.
> 
> e.g.
> image
> 
> Untested:
> 
> ```css-less
> div.PendingRegistrations {
>     thead > tr > th > label, tbody > tr > td > label {
>         margin-bottom: 0
>         height: 17px;
>     }
> }
> ```

Done thank you!

**Review by luke- — APPROVED** [review]

**Pull request merged**